templates/suitecrm/index.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Output,
+  randomPassword,
+  randomString,
+  Services,
+} from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const mariaPassword = randomPassword();
+  const appKey = Buffer.from(randomString(32)).toString("base64");
```

Comment:
Isn't `randomString` enough here?

Reply:
Checked myself, needs to be this specific to work